src/App.js

```diff
@@ -15,21 +15,40 @@ chance.mixin({
 });
 
 const allFriends = Array.from({length: 20}, () => chance.friend());
-
-console.log(allFriends);
+const bestFriends = allFriends.filter(friend => friend.isBestFriend);
 
 export default class App extends Component {
   render() {
     return (
       <div className="App">
+        <PeopleFilteredByProp array={allFriends} arrayTitle="all friends"/>
+        <PeopleFilteredByProp array={bestFriends}  arrayTitle="best friends"/>
```

Comment:
might be a better idea to be a bit more clear about naming of props, instead of `array` - which a linter might complain about (lots of linters complain if you name things array, object, string etc) - name it 'people' - which is more semantically in line with what the data is.